### PR TITLE
Component Registry project higlights and adding DevCon24 Call for presentations proposal

### PR DESCRIPTION
**Pull request by VojtechBartoska**
- adding CFP graphic and text for DevCon24
- fixed wording for Component Registry

### CODE REVIEW

**Comment by VojtechBartoska** [comment]
cc @pedrominatel and @kumakey

File: profile/README.md [hunk 1]
```diff
@@ -10,14 +10,15 @@ Below you can find a selection of Espressif’s open-source projects. Our full r
...
+    <img src="profile/Logo_white_Component_Registry.png" style="width: 50%; height: auto">
 </div>
 
-###  **ESP Registry**
+###  **ESP Component Registry**
 
-The **ESP Registry** is a tool that downloads dependencies for any ESP-IDF CMake project. It makes sure that the right versions of all components required for a successful build of your project are in place. The download happens automatically during a run of CMake.
+The **ESP Component Registry** is a tool for components that can be used with the ESP-IDF and provides a convenient way for developers to discover and download components for their projects.
+
+Find the most exciting ESP-IDF components on [ESP Component Registry](https://components.espressif.com/)!
```
Comment (kumekay) [inline profile/README.md:20]: The description is for the CLI tool, not for the registry

```suggestion
The **[ESP Component Registry](https://components.espressif.com/)** allows you to discover, download, and publish components and examples for the ESP-IDF.
```

File: profile/README.md [hunk 2]
```diff
@@ -10,14 +10,15 @@ Below you can find a selection of Espressif’s open-source projects. Our full r
...
 
-Find the most exciting ESP-IDF components on [ESP Registry](https://components.espressif.com/)!
 If you want to contribute to this project, you can do so at [IDF Component Manager repository](https://github.com/espressif/idf-component-manager).
```
Comment (kumekay) [inline profile/README.md:22]:
```suggestion
If you have any issues or suggestions about the ESP Component Registry, feel free to open an issue in the [IDF Component Manager repository](https://github.com/espressif/idf-component-manager).

```

**Review by kumekay — COMMENTED** [review]
@VojtechBartoska  Thank you for fixing this, I also corrected the wording.

**Comment by pedrominatel** [comment]
LGTM. @VojtechBartoska can you commit the changes?

**Comment by pedrominatel** [comment]
> LGTM. @VojtechBartoska can you commit the changes?

@VojtechBartoska, please check if the logo has a transparent background. It looks like the logo has a grey bg.

**Comment by VojtechBartoska** [comment]
Done, @pedrominatel can you please help merging?

**Comment by kumekay** [comment]
@VojtechBartoska Why do we need to change the ESP Component Registry logo? The current one looks fine. https://github.com/espressif/.github/raw/main/profile/esp_registry_logo.png

**Comment by VojtechBartoska** [comment]
@kumekay issue with rendering white/dark theme a browser

**Comment by pedrominatel** [comment]
@VojtechBartoska, the logo was fixed on this PR: https://github.com/espressif/.github/pull/11

File: Logo_white_Component_Registry.png
Comment (pedrominatel) [inline Logo_white_Component_Registry.png:1]: Revert this logo. The right one was merged on the PR https://github.com/espressif/.github/pull/11

File: profile/README.md [hunk 3]
```diff
@@ -7,18 +7,30 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
 
 Below you can find a selection of Espressif’s open-source projects. Our full repository list can be found [here](https://github.com/orgs/espressif/repositories).
 
+## Call for Presentations - DevCon24
+
+Third edition of Online Developer’s Conference DevCon24 will take place on September 3-5, 2024.
```
Comment (pedrominatel) [inline profile/README.md:12]: The third edition of Online...

File: profile/README.md [hunk 4]
```diff
@@ -7,18 +7,30 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
...
+
+We are now collecting talks proposals, so don't miss this opportunity to showcase your interesting projects.
+
+We are open to any proposal, Maker’s projects are very welcomed!
```
Comment (pedrominatel) [inline profile/README.md:16]: I think we can remove this sentence.

**Review by pedrominatel — COMMENTED** [review]
PTAL

File: Logo_white_Component_Registry.png
Comment (VojtechBartoska) [reply Logo_white_Component_Registry.png:1]: reverted

**Comment by VojtechBartoska** [comment]
@pedrominatel all comments fixed, please squash the commits

**Review by pedrominatel — APPROVED** [review]
LGTM

**Pull request merged**